cmake/environmentchecks.cmake

```diff
@@ -128,6 +130,10 @@ else()
   endif()
 endif()
 
+if(BUILDCACHE_EXE)
+  set(BUILDCACHE_AVAILABLE TRUE CACHE INTERNAL "Executable required to cache compilations.")
+endif()
+
 if(WIN32)
```

Comment:
given the way its used, shouldn't this not bother checking for `clcache` and `ccache` if `buildcache` is found?

Reply:
Fair point, done in commit adb22bde.